tasks/freeradius-mysql.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Freeradius additional mysql system packages
+  when: openwisp2_radius and freeradius_sql.dialect == "mysql"
+  apt:
+    name:
+      - freeradius-mysql
+      - mariadb-server
```

Comment:
installing MySQL server is out of scope for this role and should be done with another role which can handle its configuration.
It's not assured the DB will run on the same host.
Keep also n mind MariaDB is not MySQL although it was forked from it, so if mysql is specified, mysql should be used, mariadb has different DB drivers.

Reply:
Removed